zndraw/draw/__init__.py

```diff
@@ -123,6 +123,15 @@ class Rhomboid(Object3D):
     vectorC: t.Tuple[float, float, float] | list[float] = (0, 0, 10)
 
 
+class Custom2DShape(Object3D):
+    points: (
+        t.Tuple[t.Tuple[float, float], ...]
+        | list[t.Tuple[float, float]]
+        | list[list[float]]
+        | t.Tuple[list[float], ...]
+    )
```

Comment:
Could we make this only `list[tuple[float, float]]`? All the other type hints are a bit confusing and e.g. `list[list[float]]` might not be correct if the points are only 2D?

Reply:
Sure